menu/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from rest_framework import viewsets, filters
 
-# Create your views here.
+from django_filters.rest_framework import DjangoFilterBackend
+
+from menu.filters import CustomSearchFilter, CustomRangeFilter
+from menu.serializers import MenuSerializer
+from product.models import Product
+
+
+class MenuView(viewsets.ReadOnlyModelViewSet):
+    serializer_class = MenuSerializer
```

Comment:
new serializer

Reply:
done in to another view - LocationMenuView